chartfx-chart/src/main/java/de/gsi/chart/viewer/DataView.java

```diff
@@ -72,21 +78,31 @@ public void set(final DataViewWindow newNode) {
         }
     };
 
-    public DataView(final String name, final Node icon) {
+    @Deprecated // is this default constructor really needed?
```

Comment:
Needed only if we want to allow only giving name or icon. I added this, when I was not yet fully aware of the intended use of the DataView, i think we should remove them before merge to keep the api clean.

Reply:
incorporated with commit 05ba35673e7f8fd98fb620daf0dc086d874e5584